### PR TITLE
add link between https warning

### PR DESCRIPTION
**Pull request by bahmutov**
to make https://github.com/cypress-io/cypress/discussions/16033 more discoverable in our docs

### CODE REVIEW

File: content/faq/questions/using-cypress-faq.md [hunk 1]
```diff
@@ -698,7 +698,9 @@ Yes! Check out our [ESLint plugin](https://github.com/cypress-io/eslint-plugin-c
 
 ## <Icon name="angle-right"></Icon> When I visit my site directly, the certificate is verified, however the browser launched through Cypress is showing it as "Not Secure". Why?
 
-This is normal. Cypress modifies the traffic between your server and the browser. The browser notices this and displays a certificate warning. However, this is purely cosmetic and does not alter the way your application under test runs in any way, so you can safely ignore this warning.
+When using Cypress to test an https site, you might see a browser warning next to the browser URL. This is normal. Cypress modifies the traffic between your server and the browser. The browser notices this and displays a certificate warning. However, this is purely cosmetic and does not alter the way your application under test runs in any way, so you can safely ignore this warning. The network traffic between Cypress and the backend server still happens via https protocol.
```
Comment (Laffini) [inline content/faq/questions/using-cypress-faq.md:701]: Very minor suggestion, however `https` should be HTTPS.

**Review by Laffini — COMMENTED** [review]
Extremely minor suggested change.

File: content/faq/questions/using-cypress-faq.md (hunk 1 above)
Comment (flotwig) [reply content/faq/questions/using-cypress-faq.md:701]:
```suggestion
When using Cypress to test an HTTPS site, you might see a browser warning next to the browser URL. This is normal. Cypress modifies the traffic between your server and the browser. The browser notices this and displays a certificate warning. However, this is purely cosmetic and does not alter the way your application under test runs in any way, so you can safely ignore this warning. The network traffic between Cypress and the backend server still happens via HTTPS.
```

**Review by flotwig — APPROVED** [review]

**Review by Laffini — APPROVED** [review]
No further comments.

**Pull request merged**